src/functions/moveChecks/bishopCheck.ts

```diff
@@ -0,0 +1,42 @@
+import { diagonalRowNE, diagonalRowSW,rowFinder,column, firstColumn, columnFinder, row, lastColumn, lastRow, firstRow, diagonalRowSE, diagonalRowNW } from "../../global"
+import { allowedSquares } from "../../stores"
+
+export const bishopCheck = (targetSquare:number) => {
+
+const tempArray = []
+const startSquare = targetSquare
```

Comment:
Looks like you are not using this? Also, if you wanted to call it startSquare, you cold have done it in the arguments?